src/export_from_inboveg/HT3260_data_preparation.Rmd

```diff
@@ -0,0 +1,420 @@
+# values < 1% are converted to 0.5 % (via value_perc)
+```
+
+controle - tmp
+```{r, eval=FALSE}
+layerqual_3260 %>% 
+  select(value, value_perc) %>% 
+  arrange(value) %>% distinct()
+# to do (opvolgen): 0,1 al aangepast in inboveg door Pieter, want levert NA op
+
+```
+
+
+### combined
+= site_characteristics = structure + layer
+```{r}
+site_characteristics <- bind_rows(structure, layerqual) %>%
+    mutate(var = str_to_lower(var),
+                 value_class = str_to_lower(value_class))
+  
+```
+
+
+## vegetation
+rearrangement of vegetation
+```{r}
+vegetation <- vegetation_3260 %>% 
+  mutate(name = ifelse(is.na(name_scientific), name_original, name_scientific)) %>% 
+  select(survey, recording_givid, name, phenology_code, species_cover_code, species_cover, scale) %>% 
+  distinct() # tmp - to do opvolgen of dubbels eruit zijn
+
+# to do (later): nagaan of Nederlandse naam beschikbaar is in inboveg (@ Sofie of Pieter) 
+
+```
+
+controle - tmp
+```{r, eval=FALSE}
+# lijst my taxa 
+vegetation_3260 %>% 
+  filter(is.na(name_scientific)) %>% 
+  count(name_original) %>% 
+  arrange(name_original)
+
+sel_species <- c("Enteromorpha sp.")
+sel_species <- c("Salix 'alba' groep")
+sel_species <- c("Poaceae", "poaceae")
+
+vegetation_3260 %>% 
+  filter(name_original %in% sel_species) %>% 
+  left_join(select(header, recording_givid, user_reference), by = "recording_givid") %>% 
+  select(recording_givid, user_reference, name_original, species_cover_code) 
+  
+# soorten van 'my taxa' uniformiseren want identiek: to do (opvolgen; @ Pieter gevraagd)
+#   Callitriche 'obtusangula/platycarpa' groep en Callitriche platycarpa/obtusangula   -
+#   darmwier   en Enteromorpha sp.
+# gras (poaceae, vegetatief, niet gedetermineerd),   gras vegetatief en Poaceae, poaceae
+# mos en mos niet gedetermineerd
+# Salix 'alba' groep en Salix smalbladig en wilg smal
+
+```
+
+
+# data controle 
+##  input
+```{r, eval = FALSE}
+this_sample <- "Dommel_14"
+
+header_sel <- header_3260 %>% filter(str_detect(user_reference, this_sample))
+
+classif_sel <- classif_3260  %>% semi_join(header_sel, by = "recording_givid")
+structure_sel <- structure_3260  %>% semi_join(header_sel, by = "recording_givid")
+sitequal_sel <- sitequal_3260  %>% semi_join(header_sel, by = "recording_givid")
+layerqual_sel <- layerqual_3260  %>% semi_join(header_sel, by = "recording_givid")
+vegetation_sel <- vegetation_3260  %>% semi_join(header_sel, by = "recording_givid")
+
+# is ok; alles zit erin
+
+```
+
+##  output
+
+```{r, eval = FALSE}
+header_sel <- header %>%  
+  filter(str_detect(user_reference, this_sample))
+
+site_characteristics_sel <- site_characteristics %>% semi_join(header_sel, by = "recording_givid")
+vegetation_sel <- vegetation %>% semi_join(header_sel, by = "recording_givid")
+
+# is ok; alles zit erin
+
+```
+
+# data export
+
+```{r}
+write_vc(header, "HT3260_header", 
+         root = path, sorting = c("recording_givid"))
```

Comment:
Als je variabelen toevoegt of namen van variabelen verandert, moet je bij write_vc steeds argument strict = FALSE toeveogen anders krijg een error

```suggestion
write_vc(header, "HT3260_header", 
         root = path, sorting = c("recording_givid"), strict = FALSE)
```

Reply:
ok; aangepast